jme3-core/src/main/java/com/jme3/anim/AnimLayer.java

```diff
@@ -79,9 +81,13 @@ public class AnimLayer implements JmeCloneable {
     /**
      * The name of this layer.
      */
-    final private String name;
+    private String name;
 
     private boolean loop = true;
+    
+    protected AnimLayer() {
```

Comment:
please add the javadoc to the empty constructor as below:

    /**
     * For serialization only. Do not use.
     */